documentation/src/docs/security/securing-distributed-mode-cluster.md

```diff
@@ -310,106 +103,134 @@ Controller services can be configured in three different ways:
       ...          
    ```
 
-The following table lists the Controller's TLS and auth parameters and representative values, for quick reference. For a detailed description of these parameters, refer to [this](https://github.com/pravega/pravega/blob/master/documentation/src/docs/security/pravega-security-configurations.md) document.
+The following table lists the Controller service's TLS and auth parameters as well as samples values, for quick reference. 
+For a detailed description of these parameters, refer to 
+[this](https://github.com/pravega/pravega/blob/master/documentation/src/docs/security/pravega-security-configurations.md) document.
```

Comment:
"refer to [this]" -> "refer to the [Pravega Security Configurations]"

Reply:
Done.